packages/fcl/src/exec/utils/prep-template-opts.js

```diff
@@ -2,7 +2,7 @@ import {retrieve} from "../../document/document.js"
 import {normalizeInteractionTemplate} from "../../normalizers/interaction-template/interaction-template"
 import {deriveCadenceByNetwork} from "../../interaction-template-utils/derive-cadence-by-network.js"
 import {deriveDependencies} from "./derive-dependencies"
-import {isString} from "./is"
+import {isString} from "../../utils/is.js"
```

Comment:
```suggestion
import {isString} from "../../utils/is"
```
maybe we could TS-proof it just in case we don't notice later